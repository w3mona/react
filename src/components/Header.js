import React from 'react';
import {NavLink} from 'react-router-dom';

const Header=()=>(
  <header>
      <h1>Expensify</h1>
      <NavLink to="/" activeClassName="isActive"  exact={true} >Home</NavLink> 
      <NavLink to="/create" activeClassName="isActive" >Create</NavLink> 

      <NavLink to="/help" activeClassName="isActive">Help</NavLink> 
  </header>
);


export default Header;