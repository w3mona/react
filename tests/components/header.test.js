import React from 'react';
import Header from '../../src/components/Header';
//import ReactShallowRenderer from 'react-test-renderer/shallow';
import toJSON from 'enzyme-to-json';
import {shallow} from 'enzyme';

test("test header component",()=>{
    const wrapper = shallow(<Header/>);

    expect(toJSON(wrapper)).toMatchSnapshot();

   // expect(wrapper.find('h1').text()).toBe('Expensify');


    /*
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header/>);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
    */


})