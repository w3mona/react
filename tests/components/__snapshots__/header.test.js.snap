// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test header component 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    to="/create"
  >
    Create
  </NavLink>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
